BUILD

```diff
@@ -67,9 +67,6 @@ cc_library(
     ],
     includes = ["include"],
     linkopts = select({
-        "//:osx": [
-            "-framework Foundation",
-        ],
         "//:ios": [
             "-framework Foundation",
         ],
```

Comment:
Does any of the same logic apply to "ios"?

Reply:
This hasn't caused a problem for Envoy Mobile, and we do iOS builds